rune/libenclave/internal/runtime/pal/skeleton/liberpal-skeleton.c

```diff
@@ -576,6 +610,9 @@ int __pal_create_process(pal_create_process_args *args)
 		return -1;
 	}
 
+	if (backend_kvm)
```

Comment:
Is it possible to load an application here?

Reply:
e.g, set the init process through kernel command line.

Reply:
At present, the init process to be run can be specified by `runtime.args`, which is used to replace the default `/sbin/init`.